Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -8693,6 +8749,28 @@ a_Player:OpenWindow(Window);
 					},
 					Notes = "Sets the custom name visiblity of this monster. If it's false, you only see the name when you sight the mob. If it's true, you always see the custom name.",
 				},
+				SetLeashedTo =
+				{
+					Params =
+					{
+						{
+							Name = "Entity",
+							Type = "cEntity",
+						},
+					},
+					Notes = "Leash the monster to a entity.",
+				},
+				SetUnleashed =
+				{
+					Params =
+					{
+						{
+							Name = "ShouldDropPickup",
```

Comment:
Personally I'd call it `leash`, because `lead` can mean other things, too (are there mods that add other ores, such as lead, into the game?)
After all, we have `planks` and `log` and no `wood`, because it caused confusion, and everyone was happy with it :)